react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.js

```diff
@@ -85,6 +85,12 @@ export default class JitsiStreamBackgroundEffect {
      * @returns {void}
      */
     runPostProcessing() {
+
+        const { height, width } = this._stream.getVideoTracks()[0].getSettings()
```

Comment:
Can you use a const track instead of calling `this._stream.getVideoTracks()` twice ?

Reply:
Done